src/views/Clients/ViewClients/index.jsx

```diff
@@ -31,9 +36,28 @@ import clientsQuery from './clients.graphql';
 }))
 export default class ViewWorker extends PureComponent {
   state = {
-    clientSearch: '',
+    clientSearch: this.props.user ? this.props.user.credentials.clientId : '',
+    clientId: this.props.user ? this.props.user.credentials.clientId : '',
```

Comment:
Given that we're not using `getDerivedStateFromProps` which will run when the site is first loaded, we can do:

```jsx
state = {
  clientSearch: '',
  previousClientId: '',
};
```